Cargo.toml

```diff
@@ -26,15 +27,16 @@ license = "Unlicense/MIT"
 description = "Boa is a Javascript lexer, parser and compiler written in Rust. Currently, it has support for some of the language."
 
 [workspace.dependencies]
+boa_ast = { version = "0.16.0", path = "boa_ast" }
 boa_engine = { version = "0.16.0", path = "boa_engine" }
-boa_interner = { version = "0.16.0", path = "boa_interner" }
 boa_gc = { version = "0.16.0", path = "boa_gc" }
-boa_profiler = { version = "0.16.0", path = "boa_profiler" }
-boa_unicode = { version = "0.16.0", path = "boa_unicode" }
+boa_icu_provider = { version = "0.16.0", path = "boa_icu_provider" }
+boa_interner = { version = "0.16.0", path = "boa_interner" }
 boa_macros = { version = "0.16.0", path = "boa_macros" }
-boa_ast = { version = "0.16.0", path = "boa_ast" }
 boa_parser = { version = "0.16.0", path = "boa_parser" }
-boa_icu_provider = { version = "0.16.0", path = "boa_icu_provider" }
+boa_profiler = { version = "0.16.0", path = "boa_profiler" }
+boa_runtime = { version = "0.16.0", path = "boa_runtime" }
+boa_unicode = { version = "0.16.0", path = "boa_unicode" }
```

Comment:
Here I just re-ordered everything alphabetically, as we have them in the top, and added the boa_runtime.